src/chi-vue/src/constants/classes.ts

```diff
@@ -0,0 +1,46 @@
+/* Common */
+export const activeClass = '-active';
```

Comment:
Consts should be UPPER_CASE.

Reply:
If you define them as readonly or as consts you can include them in types as well: 
```TypeScript 
export const CLOSED_CLASS = '-closed' as const;
```
This doesn't mean you need to put as const in every const you use, but it might be useful (mana) when you start trying doing magic with types.